### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by umangachapagain**
Signed-off-by: Umanga Chapagain <chapagainumanga@gmail.com>

### CODE REVIEW

**Review by aruniiird — APPROVED** [review]
LGTM... (=

File: README.md [hunk 1]
```diff
@@ -1,25 +1,42 @@
-# Operator for storage monitoring mixin
+# Operator for Storage Monitoring Mixins
 
-This operator takes care of deployment of mixins of storage system in
-kubernetes. The mixins is an abstraction framework that makes it easier
+Monstorak operator takes care of deployment of monitoring mixins for storage systems in
+K8s/Openshift.
```
Comment (cloudbehl) [inline README.md]: mentioned k8s but isn't going to work.

k8s doesn't have the command: https://github.com/monstorak/monstorak/pull/35/files#diff-04c6e90faac2675aa89e2176d2eec7d8R16

Comment (aruniiird) [reply README.md]: @cloudbehl ,
 this can be treated as a separate bug and should be fixed. 'README' still can be same...

**Review by shtripat — COMMENTED** [review]
LGTM
If we are using `oc` lets mention in README at some stage that it works with OKD.

File: README.md (hunk 1 above)
Comment (umangachapagain) [reply README.md]: fixed.

**Comment by umangachapagain** [comment]
> LGTM
> If we are using `oc` lets mention in README at some stage that it works with OKD.

Fixed.

**Review by shtripat — APPROVED** [review]
LGTM

**Review by aruniiird — APPROVED** [review]
Approved

**Pull request merged**